admin/app/conf/AdminConfiguration.scala

```diff
@@ -42,12 +42,13 @@ object AdminConfiguration {
     lazy val appName = configuration.getStringProperty("api.dfp.applicationName")
   }
 
-  lazy val oauthCredentials: Option[OAuthCredentials] =
+  lazy val oauthCredentials: Option[OAuthCredentialsWithMultipleCallbacks] =
```

Comment:
I went the other way and instead of having 2 properties and then reassemble the callback url, I now store in the config a list of full callback urls. Tell me what you think